.golangci.yaml

```
@@ -83,15 +83,15 @@ linters-settings:
       - simmilar
   perfsprint:
     # Optimizes even if it requires an int or uint type cast.
-    int-conversion: false
+    int-conversion: true
     # Optimizes into `err.Error()` even if it is only equivalent for non-nil errors.
     err-error: true
     # Optimizes `fmt.Errorf`.
     errorf: true
     # Optimizes `fmt.Sprintf` with only one argument.
-    sprintf1: false
+    sprintf1: true
     # Optimizes into strings concatenation.
-    strconcat: false
+    strconcat: true
```

I personally prefer not to use this optimization.

```suggestion
    strconcat: false
```

@DmitriyLewen @simar7 @nikpivkin However, if you prefer string concatenation, I'm okay with that.

---

I would also vote not to use this. `fmt.Sprintf` seems fine to me. Is this subjective or there's something on the table w.r.t. performance by using this?

---

sometimes we use a long strings, so string concatenation will be almost unreadable.
So I think it is better to use `fmt.Sprintf`.

---

I think the performance gain will be imperceptible, but the readability will be lost. If we find a weakness, we can improve it.